docs/src/operate/c8y/supported-operations.md

```diff
@@ -377,16 +383,64 @@ te/device/<name>///cmd/command/c8y-mapper-1800380
 
 ```json5 title="Payload"
 {
-      "status": "init",
-      "text":"echo helloworld",
-      "foo":"bar",
+  "status": "init",
+  "text":"echo helloworld",
+  "foo":"bar"
 }
 ```
 
 Make sure that provided input is JSON object, otherwise the operation execution will be skipped.  
 
 :::
 
+:::info
+
+The `output` field is required for specific operations (e.g., `c8y_Command` and `c8y_RelayArray`) that need to parse the operation result upon successful execution. This field accepts either a `String` or an `Array` value. Placeholders in the output will be replaced with the actual values and appended as parameters to the SmartREST `503`/`506` messages.
+
+#### Example 1: c8y_Command
+
+For the `c8y_Command` operation, the execution output is stored in the `result` field. Here is an example JSON payload when it is successful:
+```json5 title="c8y_Command"
+{
+  "c8y-mapper": {
+    "on_fragment": "c8y_Command",
+    "output": "${.payload.result}"
+  },
+  "command":"echo helloworld",
+  "result":"helloworld",
+  "status": "successful"
+}
+```
+
+This message is converted to the following SmartREST message:
+```
+503,c8y_Command,helloworld
+```
+
+#### Example 2: c8y_RelayArray
+
+For the c8y_RelayArray operation, the output is an array. Here is an example JSON payload when it is successful:
+
+```json5 title="c8y_RelayArray"
+{
+  "c8y-mapper": {
+    "on_fragment": "c8y_RelayArray",
+    "output": "${.payload.result}"
+  },
+  "command":"echo helloworld",
+  "result":["OPEN","CLOSED"],
```

Comment:
```suggestion
  "states":["OPEN","CLOSED"],
```
May be we can use a different key in this second example to convey the point that it doesn't always have to be named `result`.